lib/megatest/cli.rb

```diff
@@ -92,7 +92,7 @@ def run_tests
         end
       end
 
-      @config.selectors = Selector.parse(@argv)
+      @config.selectors = Selector.new(@config).parse(@argv)
```

Comment:
Ah, I just realized that this broke the feature I wanted. The `@config` is loaded after it is copied into the selector loader so any change to `test_globs` aren't applied.

So we need to change the order so `load_config` happens before the selector is set, which actually make impossible to set a custom selector.